tests/helpers/resize-container.js

```diff
@@ -5,9 +5,9 @@ import { find } from '@ember/test-helpers';
 // This is a magic number. It is the time (in ms) for things to `settle`
 // after a resize. It is the time that we need to wait before assertions
 // that should pass will always pass.
-const RERENDER_TIME = 100;
+const RERENDER_TIME = 50;
```

Comment:
The new value `50` matches that used in tests for `ember-on-resize-modifier`.

https://github.com/PrecisionNutrition/ember-resize-kitchen-sink/blob/v0.4.5/packages/ember-on-resize-modifier/tests/utils.js#L3